libzerocash/ZerocashParams.h

```diff
@@ -24,29 +24,34 @@ class ZerocashParams {
 public:
     typedef default_r1cs_ppzksnark_pp zerocash_pp;
 
-	ZerocashParams(const unsigned int tree_depth);
-	ZerocashParams(zerocash_pour_proving_key<zerocash_pp>* p_pk_1,
-                   zerocash_pour_verification_key<zerocash_pp>* p_vk_1);
-
-    ZerocashParams(const unsigned int tree_depth,
-                   std::string pathToProvingParams,
-                   std::string pathToVerificationParams);
+    ZerocashParams(
+        const unsigned int tree_depth,
```

Comment:
[does not block] Since the ``tree_depth`` and parameters are mutually dependent, it would be nice in the future to derive the ``tree_depth`` from the parameters (even if that requires merely including them in the serialization).